### PR TITLE
releasing the FB pod pin

### PR DESCRIPTION
**Pull request by bsz0206**

### CODE REVIEW

File: ios/flutter_login_facebook.podspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Login via Facebook for Flutter projects.
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*'
   s.dependency 'Flutter'
-  s.dependency 'FBSDKLoginKit', '16.0.1'
+  s.dependency 'FBSDKLoginKit', '~> 16.0.1'
```
Comment (Saifallak) [inline ios/flutter_login_facebook.podspec:18]:
```suggestion
  s.dependency 'FBSDKLoginKit', '~> 16.1'  
```

**Review by Saifallak — CHANGES_REQUESTED** [review]

**Comment by Saifallak** [comment]
can we merge it to support with facebook app events ? https://github.com/oddbit/flutter_facebook_app_events/pull/309

**Comment by lcsvcn** [comment]
+1

**Comment by lcsvcn** [comment]
@Saifallak I tested out your implementation is still telling:

```
 FBSDKLoginKit (~> 16.0.1) was resolved to 16.0.1, which depends on
        FBSDKCoreKit (= 16.0.1)
```
Screenshot 2024-04-16 at 09 09 51

**Comment by greymag** [comment]
Version 17.0 requires support of the Limited login feature. See #108 for details. I'm closing this PR for now.

**Pull request closed without merging**